include/zephyr/arch/riscv/common/linker-rom.ld

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2016-2017 Jean-Paul Etienne <fractalclone@gmail.com>
```

Comment:
now we have `common/linker-rom.ld` but also a `linker/common-rom.ld`.

this is bound to create confusion as to when something is supposed to go into `linker/common-rom` and when to go into `common/linker-rom`.

Could you please improve on location and or naming of files.

Reply:
I can give the naming some thought, but I think `arch/riscv/common/linker-rom.d` is unlikely to be confused with `zephyr/linker/common-rom.ld`; the full path will be present in most users. However the `common` here does seem like it can be improved on.